eks/core.py

```diff
@@ -667,7 +743,7 @@ def compute_covariance_matrix(ensemble_preds):
     cov_mats = []
     for i in range(n_keypoints):
         E_block = extract_submatrix(E, i)
-        cov_mats.append(E_block)
+        cov_mats.append([[1, 0], [0, 1]])
```

Comment:
is E_block unsued now?

Reply:
in that case can you also comment out E_block, and say that using it instead of the identity matrix would lead to a correlated dynamics model, but this was leading to negative variance values and has not yet been debugged